ingest/dense.py

```diff
@@ -74,6 +97,9 @@ def transform(self):
             self.info_logger.info(
                 f'Transforming gene :{gene}', extra=self.extra_log_params
             )
+            if gene in self.gene_names:
+                raise ValueError(f'Duplicate gene: {gene}')
```

Comment:
Will raising ValueError look like one of our scripts failed and exited?
Should this error be logged so the user receives information in the ingest notification email about why their parse failed?

Reply:
The error 'Duplicate gene: {gene}' is passed to the log file and it presented in the email users get for parsing. Parsing fails when the duplicate gene is encountered.  I think that is the desired behavior.